### PR TITLE
Backport 6123 for release branch

### PR DESCRIPTION
**Pull request by conorsch**
## Status

Ready for review

## Description of Changes

Backports #6123 to release/2.1.0 branch.

## Testing
- [ ] Commit is identical to that in #6123
- [ ] Current release branch is targeted by the PR

### CODE REVIEW

**Comment by conorsch** [comment]
N.B. I intentionally used `git cherry-pick -x` in order to get a citation for the source commit into the message. Helps with review, to see how the commits are related. We always use cherry-pick-x for backports, which is documented here https://docs.securedrop.org/en/stable/development/release_management.html?highlight=cherry-pick#pre-release

**Comment by conorsch** [comment]
> I intentionally used git cherry-pick -x

Eep, looks like I _didn't_ do that, @cfm. My mistake, let me try again...

**Comment by conorsch** [comment]
Voila! I am now practicing what I am preaching once more.

**Review by cfm — APPROVED** [review]
Approved for merge (once CI passes) per test plan:

- [x] Commit is identical to that in #6123
    ```sh-session
    (.env) cfm@ozymandias{1014}$ git checkout backport-6123-to-release-branch 
    Already on 'backport-6123-to-release-branch'
    Your branch is up to date with 'origin/backport-6123-to-release-branch'.
    (.env) cfm@ozymandias{1015}$ git cherry-pick -n origin/qa-update-playbook 
    (.env) cfm@ozymandias{1017}$ git diff | wc -l    [~/work/securedrop 12:49]
           0
    ```
- [x] Current release branch is targeted by the PR = `release/2.1.0`
